### PR TITLE
added tx_new_surge

### PR DESCRIPTION
**Pull request by Iam-Wangereka**

### CODE REVIEW

File: .idea/compiler.xml
Comment (Rugute) [inline .idea/compiler.xml:1]: Never send this again

File: .idea/encodings.xml
Comment (Rugute) [inline .idea/encodings.xml:1]: Remove from this PR

**Review by Rugute — APPROVED** [review]
Done

File: src/main/java/ampath/or/ke/spot/models/RRI.java
Comment (Rugute) [inline src/main/java/ampath/or/ke/spot/models/RRI.java:1]: Good job

**Comment by Iam-Wangereka** [comment]
Done

**Pull request closed without merging**